creds.py

```diff
@@ -0,0 +1,11 @@
+# TELEGRAM
+BOT_TOKEN = ""
```

Comment:
Don't store your credentials in source code. Even if you think you'll remember to delete it before committing- you'll eventually forget. There's two alternatives:

- Store it in a special file, which is included in the `.gitignore` and thus not tracked by git. Read this file for the token
- Store it in an environment variable, and read that with `os.environ["VAR_NAME"]`. 

I personally prefer the second option. There's also python projects (dotenv) that allow you to set the environment variables from a special `.env` file, and then gitignore that (combining the two options)